pom.xml

```diff
@@ -104,6 +111,16 @@
     <build>
         <pluginManagement>
             <plugins>
+                <plugin>
+                    <groupId>org.lfenergy.compas.xsd</groupId>
+                    <artifactId>compas-scl-xsd</artifactId>
+                    <version>${compas-scl-xsd.version}</version>
+                </plugin>
```

Comment:
It seems strange to put `compas-scl-xsd` here. Are you sure it works? Shouldn't we put the maven-dependency-plugin and all the parameters underneath here.

Reply:
It allow to define the compas-scl-xsd and scl-extension versions once for the entire projet compas-sct